README.md

```diff
@@ -162,15 +162,21 @@ In place of common selectors like class, id or attribute we can use `document.qu
     $el.nextAll($filter);
 
     // Native (optional filter function)
-    function getAllSiblings(elem, filter) {
-      var sibs = [];
-      elem = elem.parentNode.firstChild;
-      do {
-          if (elem.nodeType === 3) continue; // ignore text nodes
-          if (!filter || filter(elem)) sibs.push(elem);
-      } while (elem = elem.nextSibling)
-    return sibs;
-    }
+    function getNextSiblings(elem, filter) {
```

Comment:
Actually this return all siblings, not just 'next' siblings, can you update this?
and we prefer two space for indentation.

Reply:
use `if (nextElem === elem.nextElementSibling)` can filter prevSiblings, so we can get nextSiblings, and I compare it with jQuery, and the result is the same.